exonum-java-binding-core/rust/src/runtime/config.rs

```diff
@@ -10,8 +10,12 @@ pub struct Config {
 /// JVM configuration.
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct JvmConfig {
-    /// Whether to create JVM with `Xdebug` option or not.
-    pub debug: bool,
+    /// Additional parameters for JVM.
+    ///
+    /// Passed directly to JVM while initializing EJB runtime.
+    /// Parameters must not have dash at the beginning.
```

Comment:
What will happen if a user passes one? Do we check it anywhere and show a descriptive error message?

Reply:
Probably good catch! In case of invalid parameter JVM will return pretty descriptive error message

Reply:
It's impossible to pass a parameter with a dash (Exonum core considers it as an unknown cmd flag)